appveyor.yml

```diff
@@ -0,0 +1,24 @@
+version: 1.0.{build}
+init:
+- git config --global core.autocrlf true
+build_script:
+- cd build
+- mkdir c:\projects\keys
+- '"C:\Program Files (x86)\Microsoft SDKs\Windows\v8.1A\bin\NETFX 4.5.1 Tools\sn.exe" -k c:\projects\keys\openstacknetsdk.snk'
+- '"C:\Program Files (x86)\Microsoft SDKs\Windows\v8.1A\bin\NETFX 4.5.1 Tools\sn.exe" -k c:\projects\keys\openstack.net.net35.snk'
+- '"C:\Program Files (x86)\Microsoft SDKs\Windows\v8.1A\bin\NETFX 4.5.1 Tools\sn.exe" -k c:\projects\keys\openstack.net.net40.snk'
+- '"C:\Program Files (x86)\Microsoft SDKs\Windows\v8.1A\bin\NETFX 4.5.1 Tools\sn.exe" -k c:\projects\keys\openstack.net.net45.snk'
+- '"C:\Program Files (x86)\Microsoft SDKs\Windows\v8.1A\bin\NETFX 4.5.1 Tools\sn.exe" -k c:\projects\keys\openstack.net.netcore45.snk'
+- '"C:\Program Files (x86)\Microsoft SDKs\Windows\v8.1A\bin\NETFX 4.5.1 Tools\sn.exe" -k c:\projects\keys\openstack.net.portable-net40.snk'
+- '"C:\Program Files (x86)\Microsoft SDKs\Windows\v8.1A\bin\NETFX 4.5.1 Tools\sn.exe" -k c:\projects\keys\openstack.net.portable-net45.snk'
+# Remove -SkipKeyCheck once the signing keys are added to source control
+# Remove -NoDocs once the SHFB projects are compatible with AppVeyor builds
+- powershell -Command ".\build.ps1 -SkipKeyCheck -NoDocs`
```

Comment:
Should this have a `"` instead of a backtick at the end of the string? I noticed the build is using the "Release" solution configuration instead of the ReleaseNoDocs solution configuration, which means the -NoDocs switch is not getting enabled in the build script.